config.yml

```diff
@@ -126,15 +137,17 @@ java:
       MODD_VERSION: *MODD_VERSION
     test_config: *java_test_config
     template_vars:
-      VERSION: 8
+      VERSION:
+        - 8
       JAVA_VERSION: jdk8u242-b08-debian-slim
   "11":
     build_args:
       CI_HELPER_VERSION: *CI_HELPER_VERSION
```

Comment:
I think that we can delete the ci-helper for the java image, gitlab-ci have improved cache management, so it's not anymore usefull.
I will create a specific issue/improvement for this, so ignore my remark :)